FLM/FLM.Extend.cs

```diff
@@ -1,15 +1,25 @@
 ﻿using System;
+using System.ComponentModel;
 using Neo;
+using Neo.SmartContract.Framework;
 
 namespace flamingo_contract_staking
 {
     partial class FLM
     {
+        /// <summary>
+        /// params: message, extend data
+        /// </summary>
+        [DisplayName("Fault")]
+        public static event FaultEvent onFault;
+        public delegate void FaultEvent(string message, params object[] paras);
+
         private static void Assert(bool condition, string msg)
```

Comment:
If you update the framework, you can use just Runtime.Assert(msg)

Reply:
Isn't the latest version 3.1.0 ？

Reply:
It should be in the last version, isn't it?

Reply:
I found the function you meant. 
overload.

Reply:
Remove this method, and use only the native one

Reply:
Does Assert (bool condition, string message) have an error？

Reply:
I think there is this method in the latest version。
https://github.com/neo-project/neo-devpack-dotnet/blob/aad96be58ec6b78808c0b865a5eb0de00d1b514f/src/Neo.SmartContract.Framework/ExecutionEngine.cs#L29

Reply:
Oh, sorry. I got it wrong. I understand what you mean.